exercises/levenshtein-distance/canonical-data.json

```diff
@@ -0,0 +1,65 @@
+{
+  "exercise": "levenshtein-distance",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Calculate Levenshtein distance",
+      "cases": [
```

Comment:
I think this extra level of nesting (`cases` in `cases`) is unnecessary. How about just one level of nesting? Example, https://github.com/exercism/problem-specifications/blob/master/exercises/leap/canonical-data.json.

Alternatively, if we desire to add additional cases and that reveals a useful grouping (such as "insertions only", "deletions only", "replacements only", "all three"), then that would be a time to have multiple levels of nesting. If there is only one group of tests, we should not add unnecessary nesting.